<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_gestaoHarmonizacao</class>
 <widget class="QDialog" name="fm_gestaoHarmonizacao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Harmonização</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
background:#248;
color:#fff;
font-size:12pt;
border:none;
border-radius:10px;
}
QPushButton:hover{
	background:#fff;
	color:#248;
	font-size:12pt;
	border:none;
	border-radius:10px;
}
</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>-4</x>
     <y>-1</y>
     <width>731</width>
     <height>641</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Adcionar Harmonização</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>260</width>
       <height>242</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Descrição</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEditAddDescricao"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButtonSalvar_1">
          <property name="text">
           <string>Salvar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonCancelar">
          <property name="text">
           <string>Cancelar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Alterar Harmonização</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>321</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>135</width>
       <height>66</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Filtro</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditFiltrar"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonFiltrar">
        <property name="text">
         <string>Filtrar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>106</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Filtar por:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonIdHarmonizacao">
        <property name="text">
         <string>ID Harmonizaçao</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonIdDescricao">
        <property name="text">
         <string>Descricao</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>30</y>
       <width>260</width>
       <height>242</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Descrição</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEditUpdtDescricao"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButtonSalvar_2">
          <property name="text">
           <string>Salvar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonExcluir">
          <property name="text">
           <string>Excluir</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
